rust/abacus-base/src/settings/mod.rs

```diff
@@ -192,70 +258,24 @@ impl IndexSettings {
     }
 }
 
-/// Settings. Usually this should be treated as a base config and used as
-/// follows:
-///
-/// ```
-/// use abacus_base::*;
-/// use serde::Deserialize;
-///
-/// pub struct OtherSettings { /* anything */ };
-///
-/// #[derive(Debug, Deserialize)]
-/// pub struct MySettings {
-///     #[serde(flatten)]
-///     base_settings: Settings,
-///     #[serde(flatten)]
-///     other_settings: (),
-/// }
-///
-/// // Make sure to define MySettings::new()
-/// impl MySettings {
-///     fn new() -> Self {
-///         unimplemented!()
-///     }
-/// }
-/// ```
+/// Settings specific to a given chain.
 #[derive(Debug, Deserialize, Default)]
 #[serde(rename_all = "camelCase")]
-pub struct Settings {
-    /// The path to use for the DB file
-    pub db: String,
-    /// Port to listen for prometheus scrape requests
-    pub metrics: Option<String>,
+pub struct ChainSettings {
```

Comment:
Yeah, actually when I wrote it I thought it was the former. So yeah hmm, maybe DomainSettings? Since it does not imply a single chain but what is relevant to the domain. Will change and lmk if you have a better idea.

Reply:
happy with this!